packages/headless/src/features/insight-search/insight-search-actions.ts

```diff
@@ -382,6 +386,15 @@ function getFacetRequests<T extends AnyFacetRequest>(
   return Object.keys(requests).map((id) => requests[id]);
 }
 
+function buildConstantQuery(state: StateNeededByExecuteSearch) {
+  const activeTab = Object.values(state.tabSet || {}).find(
+    (tab) => tab.isActive
+  );
+  const tabExpression = activeTab?.expression.trim() || '';
+
+  return tabExpression;
```

Comment:
So activeTab.expression will always be a string and never undefined/null right?

Reply:
Yes absolutely👌